plugins/index.html

```diff
@@ -0,0 +1,61 @@
+---
+pagename: Plugin Manuals
+---
+{% include header.html %}
+
+  <div id="main">
+    <div id="content" class="container">
+      <div class="row">
+          <div class="span12">
+              <h1>Foreman Plugins <small>Manuals</small></h1>
+          </div>
+      </div>
+      <div class="row">
+        <div id="col1" class="offset3 span3">
+            <table class="table">
+                <thead>
+                    <tr>
+                        <td>Foreman Discovery</td>
```

Comment:
Could you try making this bold?  It might work better when there are a few plugins.